jellyfinclient/package.json

```diff
@@ -36,7 +36,7 @@
     "formik": "^1.4.2",
     "fs-extra": "7.0.0",
     "is-electron": "^2.2.0",
-    "jellyfin-apiclient": "https://github.com/jellyfin/jellyfin-apiclient-javascript.git",
+    "jellyfin-apiclient": "^1.0.0",
```

Comment:
Is there a reason for this change? Did we release the api-client?

Reply:
Yes the prebuild packages are now available from npmjs by anthonylavado